packages/commercetools/theme/nuxt.config.js

```diff
@@ -2,6 +2,16 @@ import webpack from 'webpack';
 
 export default {
   mode: 'universal',
+  render: {
```

Comment:
remember to put that in nuxt module later ;)

Reply:
I know, this is just for testing. That's why I marked this PR as draft :)